test-files/cases/network/conflicts/pap-002/Expected.osm

```diff
@@ -1845,7 +1845,7 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
-    <way visible="true" id="-4927" timestamp="1970-01-01T00:00:00Z" version="1">
+    <way visible="true" id="-4893" timestamp="2017-06-07T14:36:37Z" version="1">
```

Comment:
This is interesting that there is a timestamp here now.  And it seems as though it comes from one of the datasets.

Reply:
hmmm...didn't notice that.  not sure what's going on there.